cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/namespace/DefaultNamespaceResourceDeleter.java

```diff
@@ -86,6 +90,8 @@ public void deleteResources(NamespaceMeta namespaceMeta) throws Exception {
     preferencesService.deleteProperties(namespaceId);
     // Delete all applications
     applicationLifecycleService.removeAll(namespaceId);
+    // Delete Source Control repository configuration
+    sourceControlService.deleteRepository(namespaceId);
```

Comment:
doesn't really practically matter since namespace deletion is already subject to a bunch of race conditions, but it's a little better to delete this before deleting all the apps. This prevents the case where somebody does a pull operation after we have removed the applications, thus adding a new app that never gets removed from the DB.